act-rules-format.bs

```diff
@@ -170,41 +192,86 @@ When all expectations are true for a test target, the test target `passed` the r
   </ol>
 </div>
 
-An expectation MUST only use information available in the test aspects, and from the applicability and other expectations of the same rule. No other information should be used in the expectation. So for instance, an expectation could be "Expectation 1 is true and ...", but it can't be "Rule XYZ passed and ...". This ensures the rule is encapsulated.
+An Atomic rule expectation MUST only use information available in the [test aspects](#input-aspects), from the applicability, and other expectations of the same rule. No other information can be used in the expectation. So for instance, an expectation could be "Expectation 1 is true and ...", but it can't be "Rule XYZ passed and ...". This ensures the rule is encapsulated.
+
+
+Aggregated Rules (Composed rules only) {#aggregated-rules}
+==========================================================
+
+A composed rule uses results from atomic rules and aggregate it so that for each [test target](#output-test-target) a single [outcome](#output-outcome) can be determined. All atomic rules used in the [Aggregation definition](#aggregation-definition) MUST be listed in the composed rule. The Aggregated Rules section describes the input for composed rules, similar to how Aspects Under Test describe the input for Atomic rules.
+
+
+Aggregation Definition (Composed rules only) {#aggregation-definition}
+======================================================================
```

Comment:
the inner structure of this section being very close to the one of "Test definition", I would suggest merging these two, which would allow us to have the exact same structure outline for both "Atomic rules" and "Composed rules".